sql/updates/world/master/9999_99_99_30455_world.sql

```diff
@@ -0,0 +1,3 @@
+-- mage ice lance
+DELETE FROM spell_script_names WHERE spell_id IN (30455);
```

Comment:
Any good reason for deleting by spell ID and not by script name?

Reply:
I thought it was a safer way: could a spell use more than 1 script name? I am pretty new on these works; if it's the right way i will change ofc.

Reply:
Also, please use backticks around the DB table column names. They are usually unique, but in a very few cases, a random column name could be the same as some SQL keyword.